opentelemetry-zipkin/src/lib.rs

```diff
@@ -53,11 +64,31 @@
 //! ```
 //!
 //! ```no_run
-//! # fn main() -> Result<(), opentelemetry::trace::TraceError> {
-//! let tracer = opentelemetry_zipkin::new_pipeline()
-//!     .install_batch(opentelemetry_sdk::runtime::Tokio)?;
-//! # Ok(())
-//! # }
+//! use opentelemetry_sdk::{
+//!     runtime::Tokio,
+//!     trace::{
+//!         span_processor_with_async_runtime::BatchSpanProcessor,
```

Comment:
we don't need to use the async_runtime one in examples. BatchSpanProcessor now works without any async runtimes, and is the default already.

(I realized this need to fixed in the OTLP doc too)